tests/unit/geo.tcl

```diff
@@ -263,6 +267,12 @@ start_server {tags {"geo"}} {
         r georadius nyc -73.9798091 40.7598464 10 km COUNT 3
     } {{central park n/q/r} 4545 {union square}}
 
+    test {GEORADIUS with multiple WITH* tokens} {
```

Comment:
+ suggest adding all the combinations (order doesn't matter)

Reply:
it's a bit of an overkill no?

btw i deliberately added one with WITHCOORD WITHHASH and the same with WITHHASH WITHCOORD just to make sure that the reply order doesn't depend on the args